src/shared/2021_robot_constants.c

```diff
@@ -22,6 +22,9 @@ extern "C"
     {
         RobotConstants_t robot_constants = {
             .mass_kg = 2.5f,  // determined experimentally
+            // TODO: update units
```

Comment:
TODOs should be tagged with an issue or removed (arguably inertial factor is unitless)